src/commands/contract/deploy.ts

```diff
@@ -15,7 +13,7 @@ export class DeployContract extends SwankyCommand<typeof DeployContract> {
 
   static flags = {
     account: Flags.string({
-      required: true,
+      default: "",
```

Comment:
Try creating a static variable `DEFAULT_ACCOUNT` in SwankyCommand class and use it as default value here.
The `DEFAULT_ACCOUNT` value can be initialised during the init command constructor with `defaultAccount` from the given config parameter is any or with "alice" be default.